playbooks/06-deploy-edpm.yml

```diff
@@ -146,3 +146,13 @@
   vars:
     step: post_deploy
   ansible.builtin.import_playbook: ./hooks.yml
+
```

Comment:
why make a new play just for that?

Reply:
Seems to be the established pattern? Everything else appears to initialise a new play to execute their tasks, like HCI above:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/playbooks/06-deploy-edpm.yml#L119

Everything that is starting a new stream of work here appears to be starting a new play.

Happy to change it, but the main reason was just matching the pattern of the other jobs here.

Reply:
hm, in that case... well, there are pros and cons.

Since we probably want this new path to be called from the 06-deploy-architecture.yml, we may indeed get it in a dedicated playbook in a dedicated file (maybe named `validations.yml`?)  to avoid code duplication.

That's mostly why we'd split things in a dedicated play.

Also, if a dedicated play has to be set, please consider the pattern I'm trying to push for the ceph part as shown in #1294 PR (now merged). Namely:
in the play, add a `pre_tasks` block and check if we want or not to run the validations.

Sorry for that back'n'forth.

Reply:
Ok, so I created a validations.yml and just include_tasks from this 06-deploy-edpm.yml file. Let me know if that captures your intentions there.

Reply:
perfect!